schema/release-schema.json

```
@@ -25,6 +25,46 @@
       "format": "date-time",
       "omitWhenMerged": true
     },
+    "publisher": {
+      "title": "Publisher",
+      "description": "Information to uniquely identify the publisher of this specific release. This field is ought to be populated when a release package could contain releases from different publishers.",
```

Given that `publisher` is a property of a release, I don't think it's necessary to write 'specific release'. I added 'original' based on https://github.com/open-contracting/standard/issues/325#issuecomment-490661722.

I think it's better to omit the sentence with the condition about when the populate the field and encourage this field to be populated in all cases. Otherwise, republishers would need to add the field, in which case they would be changing a release, which should be immutable.

I added an alternative second sentence to preserve the purpose of the field and simplified the first sentence to reduce repetition.

```suggestion
      "description": "The original publisher of this release. Use this field to identify the original publisher of a release published as part of a package containing releases from different publishers.",
```